src/pages/index.js

```diff
@@ -15,86 +16,215 @@ import {
   addNewCardButton,
   profileTitleInput,
   profileDescriptionInput,
+  editAvatarButton,
+  avatarEditFormElement,
 } from "../utils/constants.js";
 
+// Api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c00d25e4-e498-4e3f-a0a9-735cb37a7bb4",
+    "Content-Type": "application/json",
+  },
+});
+
 //validators
 const editProfileFormValidator = new FormValidator(config, profileFormElement);
 editProfileFormValidator.enableValidation();
 
 const addCardValidator = new FormValidator(config, addCardFormElement);
 addCardValidator.enableValidation();
 
-//new card element
+const avatarFormValidator = new FormValidator(config, avatarEditFormElement);
+avatarFormValidator.enableValidation();
+
+//CARD FUNCTIONS
+
+//get initial cards
+let section;
+api
+  .getInitialCards()
+  .then((cards) => {
+    section = new Section(
+      {
+        items: cards,
+        renderer: (cardData) => {
+          const cardElement = createCard(cardData);
+          section.addItem(cardElement);
+        },
+      },
+      ".cards__list"
+    );
+
+    section.renderItems();
+  })
+  .catch((err) => console.error(err));
+
+//like and dislike a card
+function handleLikeClick(card) {
+  if (card._isLiked) {
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.